src/features/transactions-graph/components/horizontal.tsx

```diff
@@ -83,11 +84,25 @@ function VectorLabelText({ type }: { type: LabelType }) {
   return undefined
 }
 
+function AppCallAbiMethodName({ transaction }: { transaction: AppCallTransaction | InnerAppCallTransaction }) {
+  return (
+    <RenderInlineAsyncAtom atom={transaction.abiMethod}>
```

Comment:
For this one, we could potentially render the loading state and nothing, as the majority of circumstances we won't have app specs. Thoughts?

Reply:
As discussed, I leave this unchanged.